src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java

```diff
@@ -0,0 +1,61 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEngineering Wikia
+ */
+public class TestAdsTrackingPixels extends TemplateNoFirstLoad {
+
+  public static final ImmutableMap<String, Integer> TRACKING_PIXELS_URL_RESPONSE =
+      new ImmutableMap.Builder<String, Integer>()
+          .put("http://beacon.krxd.net/pixel.gif", 204)
+          .put("http://b.scorecardresearch.com/b", 200)
+          .put("http://pixel.quantserve.com", 200)
+          .build();
+
+  public static final ImmutableList<String> LINK_NAMES =
+      new ImmutableList.Builder<String>()
+          .add("Article1")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Article1" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)